src/systemctl/systemctl.c

```diff
@@ -3455,6 +3455,24 @@ static int exec_status_info_deserialize(sd_bus_message *m, ExecStatusInfo *i) {
         return 1;
 }
 
+typedef struct UnitCondition {
+        char *name;
+        bool trigger;
+        bool negate;
+        char *param;
+        int tristate;
```

Comment:
This structure will be full of holes on 64bit, because of alignment (char* is aligned to 8 bytes). It would be nicer to put the two char* first.